ci/build/release-prep.sh

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bash
+# Description: This is a script to make the release process easier
+# Run it with `yarn release:prep` and it will do the following:
+# 1. Check that you have a $GITHUB_TOKEN set and hub installed
+# 2. Update the version of code-server (package.json, docs, etc.)
+# 3. Update the code coverage badge in the README
+# 4. Open a draft PR using the release_template.md and view in browser
+# If you want to perform a dry run of this script run DRY_RUN=0 yarn release:prep
+
+set -euo pipefail
+
+main() {
+  if [ "${DRY_RUN-1}" -eq 0 ]; then
+    echo "Performing a dry run..."
+    # Disabled because shellcheck wants me to do this
+    # SC2209: Use var=$(command) to assign output (or quote to assign string).
+    # But neither of those match my needs
+    # We do this so that the dry run echos the commands instead of running them
+    # shellcheck disable=SC2209
+    CMD=echo
+  else
+    CMD=''
+  fi
+
+  cd "$(dirname "$0")/../.."
+
+  # Check that $GITHUB_TOKEN is set
+  if [[ -z "${GITHUB_TOKEN}" ]]; then
+    echo "We couldn't find an environment variable under GITHUB_TOKEN."
+    echo "This is needed for our scripts that use hub."
+    echo -e "See docs regarding GITHUB_TOKEN here under 'GitHub OAuth authentication': https://hub.github.com/hub.1.html"
+    exit
+  fi
+
+  # Check that hub is installed
+  if ! command -v hub &>/dev/null; then
```

Comment:
check for `gh` instead of `hub` here

Reply:
> they're not the same tool!
I may be a noob at writing scripts but I'm not THAT much of a noob.
I know they're not the same tool 😂 That's intentional! 

`gh` is `hub`'s successor, hence why I'm using it here.  Eventually, the plan is to remove the use of `hub` and only use `gh` but now, the maintainer releasing should have both because it's used [here](https://github.com/cdr/code-server/blob/main/ci/build/release-github-assets.sh#L18) and [here](https://github.com/cdr/code-server/blob/main/ci/build/release-github-draft.sh#L10). 

I'll make a follow-up issue to remove the use of `hub` and replace with `gh`.

Reply:
https://github.com/cdr/code-server/issues/2960

Reply:
Do we check for `gh` elsewhere? I'm not too sure >.<

Reply:
I think in the `update-vscode` script too since it's used there.